src/os/bluestore/BlueFS.h

```diff
@@ -453,15 +453,14 @@ class BlueFS {
 #endif
 
   int64_t _maybe_extend_log();
-  void _extend_log();
+  int64_t _extend_log(uint64_t amount);
```

Comment:
We are not using "runway" returned value anywhere.
And if needed any function can calculate it independently.